configs/ctf_score.cfg

```diff
@@ -1,10 +1,11 @@
-g_ctf_score_capture 22
-g_ctf_score_capture_assist 10
-g_ctf_score_kill 7
-g_ctf_score_penalty_drop 2
-g_ctf_score_penalty_returned 1
-g_ctf_score_penalty_suicidedrop 1
-g_ctf_score_pickup_base 1
-g_ctf_score_pickup_dropped_early 1
-g_ctf_score_pickup_dropped_late 1
-g_ctf_score_return 4
+g_ctf_score_capture 100
```

Comment:
Don't you think that these value are slightly too large, basically I predict that first week everybody would asking why they getting such large scores and then if we will back again they would ask why scores is so low 🙄.